repo-audit.yml

```diff
@@ -0,0 +1,26 @@
+# Checks configuration file. These are just some examples.
+# The order in which they are declared will be maintained.
```

Comment:
I'm not clear on what `The order in which they are declared will be maintained.` means.
Do you mean that the order in which the checks are declared is the order in which they will be applied? Is that important?